File: cs/sample/Sample/Graph/Graph.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Sample.Graph{
    public class graph{
        public int V;
        public List<int>[] adj;
        public graph(int v){
            this.V = v;
            this.adj = new List<int>[v];
            for (int i=0;i<v;i++){
                this.adj[i] = new List<int>();
            }
        }
        public void addEdge(int v,int w){
            this.adj[v].Add(w);
        }
        public void addEdges(int v,List<int> ws){
            this.adj[v] = this.adj[v].Concat(ws).ToList();
        }
    }
}

File: cpp/example/src/Graph/graph.cpp
#include "graph.h"

Graph::Graph(int V){
    this->V = V;
    this->adj = new list<int>[V];
}

void Graph::addEdge(int v,int w){
    this->adj[v].push_back(w);
}

void Graph::addEdges(int v,list<int> ws){
    list<int> tmp = this->adj[v];
    tmp.insert(tmp.end(),ws.begin(),ws.end());
    this->adj[v] = tmp;
}
